src/veh_interact.cpp

```diff
@@ -1179,10 +1179,19 @@ bool veh_interact::overview( std::function<bool(const vehicle_part &pt)> enable,
                 last = opts[idx].key;
             }
 
+            bool highlighted = false;
+            // No action means no selecting, just highlight relevant ones
+            if( pos < 0 && enable && !action ){
+                highlighted = enable(pt);
+            }
+            else if( pos == idx) {
```

Comment:
Please read doc/CODE_STYLE.md - specifically `}` and `else` on one line and spaces inside of non-empty `(` `)` and spaces between `){`. There may be more.

Reply:
Thanks for pointing that out. I had meant to set up and use astyle with the project's `.astylerc`, but ended up forgetting.